deisctl/client/client.go

```diff
@@ -56,6 +59,50 @@ func NewClient(requestedBackend string) (*Client, error) {
 	return &Client{Backend: backend}, nil
 }
 
+// Prepares a running cluster to be upgraded
+func (c *Client) UpgradePrep(argv []string) error {
+	usage := `Prepare platform for graceful upgrade.
+
+Usage:
+  deisctl upgrade-prep [options]
+`
+	_, err := docopt.Parse(usage, argv, true, "", false)
+	if err != nil {
+		return err
+	}
```

Comment:
I think deis tries to go with the 

``` go
if _, err := foo(); err != nil {
  return err
}
```
syntax when possible, though admitingly I may have introduced some of this before I knew the coding style myself. :smiley: